src/libraries/System.Runtime/tests/System.Runtime.Tests/System/Runtime/CompilerServices/RuntimeHelpersTests.cs

```diff
@@ -68,6 +68,15 @@ public static unsafe void GetObjectValue()
             Assert.Equal(i, (int)iOV);
         }
 
+        [Fact]
+        public static void EqualsTest()
+        {
+            // Boolean RuntimeHelpers.Equals(Object, Object)
```

Comment:
This should also cover comparing non-valuetypes and other paths that are special-cased in the implementation.